django_prices/forms.py

```diff
@@ -1,49 +1,72 @@
+from decimal import Decimal
+from typing import List, Optional, Tuple
 import itertools
 
+from babel.numbers import get_currency_symbol
 from django import forms
+from django.core.validators import ValidationError
 from prices import Money
 
-from .validators import (
-    MaxMoneyValidator,
-    MinMoneyValidator,
-    MoneyPrecisionValidator,
-    ValidationError,
-)
+from .utils.locale import get_locale_data
+from .validators import MaxMoneyValidator, MinMoneyValidator, MoneyPrecisionValidator
 from .widgets import FixedCurrencyMoneyInput, MoneyInput
 
 __all__ = ("MoneyField", "MoneyInput")
 
 
+def _get_symbol(currency_code: str) -> str:
+    _, locale_code = get_locale_data()
+    return get_currency_symbol(currency_code, locale_code)
+
+
+def _get_currency_choices(currencies: List[str]) -> List[Tuple[str, str]]:
+    """Generate choices for SelectField.
+    As a label we are presenting currency symbol."""
+    currency_choices = [(code, _get_symbol(code)) for code in currencies]
+    return currency_choices
+
+
 class MoneyField(forms.MultiValueField):
+    """A field for Money objects.
+
+    When only one currency is provided, it renders as FixedCurrencyMoneyInput,
+    when more you will get MoneyInput which allow you to choose both amount
+    and currency.
+
+    `available_currencies` argument is expected to be list of currency codes."""
+
     def __init__(
         self,
-        available_currencies,
-        widget=MoneyInput,
-        max_values=None,
-        min_values=None,
-        max_digits=None,
-        decimal_places=None,
+        available_currencies: List[str],
+        widget=None,
+        max_values: Optional[List] = None,
+        min_values: Optional[List] = None,
+        max_digits: int = None,
+        decimal_places: int = None,
```

Comment:
If those require dev to specify value, and are intended to be used as `kwargs`, shouldn't we move the `*args` up to force such usage from the devs?

Reply:
Was thinking about it, but decided to leave the way its now: Im afraid it would broke someones implementation and (main reason) I believe django makes same thing with their Decimal field.